aws-ts-netlify-cms-and-oauth/cms/infrastructure/index.ts

```diff
@@ -72,12 +81,14 @@ crawlDirectory(
     });
 
 // logsBucket is an S3 bucket that will contain the CDN's request logs.
-const logsBucket = new aws.s3.Bucket("requestLogs",
+const logsBucket = new aws.s3.BucketV2("requestLogs",
     {
         bucket: `${config.targetDomain}-logs`,
         acl: "private",
     });
 
+configureACL("requestLogs", logsBucket, "private");
```

Comment:
We don't need to configure anything on this bucket. By default ACLs are disabled and block public access is turned on for new buckets.